### DISCUSSION TITLE
Bumping ndarray to 0.14?

### DISCUSSION THREAD

**Discussion by matzhaugen** [vbarrielle/sprs#266] [General]
Hey, I was wondering if there is any chance you'll bump the `ndarray` dependency sometime soon, or if there are long term blockers? It will allow the ability to use some features from the `ndarray-rand` package , e.g. `sample_axis`.

**Comment by vbarrielle** [comment]
I don't think there's any blocker. We need to synchronize this update with 0.10 to respect SemVer I think.
